package.json

```diff
@@ -14,13 +14,15 @@
     "start": "node ./bin/run-multiple-apps",
     "build": "node ./bin/run-build-all"
   },
-  "devDependencies": {
-    "prettier": "^2.1.2"
-  },
   "prettier": {
+    "printWidth": 120,
     "trailingComma": "es5",
     "tabWidth": 4,
     "semi": false,
     "singleQuote": true
+  },
+  "dependencies": {
```

Comment:
devDepen

Reply:
☔️ error the root package.json contains devDependencies, this is disallowed as devDependencies vs dependencies in a private package does not affect anything and creates confusion.